src/pages/RocketsPage.jsx

```diff
@@ -1,6 +1,19 @@
-import React from 'react';
+import React, { useState } from 'react';
+import { useEffect } from 'react';
+import { useSelector } from 'react-redux';
+import { useDispatch } from 'react-redux';
```

Comment:
- [ ] I think it will be better if you import both in one line to keep your code simple.

``` import {useSelector, useDispath} from "react-redux" ```

instead of 

``` 
import { useSelector } from 'react-redux';
import { useDispatch } from 'react-redux'; 
```